Automattic/newspack-theme
# Chore: update translation files

### All Submissions:


### Changes proposed in this Pull Request:

This PR updates the theme's .pot files to include all of the strings added since the [last update](https://github.com/Automattic/newspack-theme/pull/1261).

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Spot-check a couple of the languages we support by switching to them under WP Admin > General > Settings -- supported languages include German, Spanish, Belgian French, Portuguese, Brazillian Portuguese.
3. For each language you check, make sure it's loaded on the front-end (the footer Automattic/Newspack is a good place to check). 
4. In the editor,  make sure the translation files are being loaded (the best place to check is the Featured Image options -- while all may not be translated in all languages, at least 'Large' and 'Small' should be translated. This just helps make sure the translation files are still being loaded into the editor correctly). 

### Other information:
